.github/workflows/ci.yml

```diff
@@ -148,3 +148,13 @@ jobs:
              -t mediagis/nominatim:${{ matrix.nominatim.version }} \
              -t mediagis/nominatim:${{ matrix.nominatim.version }}-${{ github.sha }} .
         working-directory: ${{ matrix.nominatim.version }}
+
+      - name: Check when REPLICATION_URL is blank
```

Comment:
Can you move this up please, so it's run before the docker build and push?

Reply:
Np, all set in latest commit.